testdata/p4_16_samples/issue1834-bmv2.p4

```diff
@@ -0,0 +1,71 @@
+/* -*- P4_16 -*- */
+#include <core.p4>
+#include <v1model.p4>
+
+struct headers { }
+
+struct metadata {
+    bit<8> test;
+}
+
+
+typedef bit<8> test_t;
+
+
+parser ParserImpl(packet_in packet, out headers hdr, inout metadata meta, inout standard_metadata_t standard_metadata) {
+    state start {
+        transition accept;
+    }
+}
+
+
+control IngressImpl(inout headers hdr, inout metadata meta, inout standard_metadata_t standard_metadata) {
+    action act(test_t a) { }    // does not work
```

Comment:
are these comments accurate?

Reply:
they were at the time the issue is filed, but now fixed.  I will update them with another commit.